Content.Server/Chat/V2/Commands/DeleteChatMessageCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Content.Server.Administration;
+using Content.Server.Chat.V2.Repository;
+using Content.Shared.Administration;
+using Robust.Shared.Toolshed;
+
+namespace Content.Server.Chat.V2.Commands;
+
+[ToolshedCommand, AdminCommand(AdminFlags.Admin)]
+public sealed class DeleteChatMessageCommand : ToolshedCommand
+{
+    [CommandImplementation("id")]
+    public void DeleteChatMessage([PipedArgument] uint messageId)
+    {
+        IoCManager.Resolve<ChatRepositorySystem>().Delete(messageId);
+    }
+}
```

Comment:
Does resolve even work for entitysystems?

Also this should just be an IEntitySystemManager dependency on the system and then you Get<System> instead as resolving is slow due to the threadlocal involved.